musicbrainz/csv/untar.py

```diff
@@ -0,0 +1,23 @@
+"""
+unzip the downloaded .tar.xz files into data/raw/extracted_jsonl
+"""
+
+import glob
+import tarfile
+
+
+def extract_file(folderpath, dest_folder):
+    """
+    untar the downloaded files
+    """
+    for filepath in glob.glob(f"{folderpath}/*.tar.xz", recursive=False):
+        with tarfile.open(f"{filepath}", "r:xz") as tar:
+            for member in tar.getmembers():
+                if member.name.startswith("mbdump"):
```

Comment:
Could you comment something about why `mbdump`? Are there other members? Why aren't they wanted? Just a line or two about that either here or in the module docstring.